components/items/ItemsGrid/ItemsGrid.vue

```diff
@@ -134,6 +134,16 @@ watch(
   }
 )
 
+watch(
+  nfts,
+  (nfts) => {
+    emit('count', nfts.length)
```

Comment:
It's not the total amount of all nfts. Please check `totalCount` in the graphql query